source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,284 @@
+
+Viewing and creating Playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select the Playbooks icon **()** in the RHS to open the Playbooks backstage and list all Playbooks associated with the current team.
+Select **+ New Playbook** to start a new Playbook and checklist. Enter a Playbook name and choose **Save Playbook**.
+
+Creating checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists are created within a Playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a Playbook, or during an incident.
+
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists can be edited during an incident as well as when not associated with an active incident.
+
+To edit a checklist, choose **Edit**.
+- Hover over an input box and select the **X** to remove the checklist item.
+- Drag the hamburger menu to rearrange the checklist items.
+
+Choose **Done** to save the changes.
+
+
+Ending incidents
+~~~~~~~~~~~~~~~~
+
+There are two ways to end an incident:
+
+- Using slash command */incident end* from within the incident channel.
+- Using the **End Incident** button in the RHS panel while in the incident channel.
+
+The incident will become inactive, be removed from the list of active incidents, and the associated channel is archived.
+
+
+Administrator's Guide
+^^^^^^^^^^^^^^^^^^^^^^
+
+Permissions
+~~~~~~~~~~~~~~~~~~~~~
+
+Incidents and Playbooks are associated with teams in Mattermost, and permissions are assigned based on the Permission scheme used in Mattermost. Incidents
+are directly tied to a specific team and channel.
+
+Public incidents
+~~~~~~~~~~~~~~~~~~~~~
+
+Public incidents are incidents where anyone with access to the team the incident channel is created on can join the incident as a participant by joining the channel
+the incident is in. Anyone on the team can join the primary channel and become an incident participant. Just like public channels, public incidents appear for everyone
+in the team they are created in.
+
+Private incidents
+~~~~~~~~~~~~~~~~~~~~~
+
+Private incidents are invite-only. Participants can be added by inviting them to the incident's primary channel. Like private channels, private incidents do not appear on
+the incident list of users who are not participants.
+
+To view incidents associated with your Mattermost team, select the **Incidents** icon in the channel header to open the RHS menu. If there are no active incidents, you can
+start one or you can view incident logs.
+
+Incidents Backstage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To open the Incidents backstage Select **Incidents & Playbooks** from the Main Menu , which lists all incidents associated with the current team. This main menu item is available to all Mattermost users.
+
+All incidents for the current team are listed for review with the following details for each incident:
+
+- Name
+- Status (**Ongoing** or **Ended**)
+- Start Date
+- End Date, if ended, otherwise **--**
```

Comment:
We just changed this to `Start time` and `End time` [here](https://github.com/mattermost/mattermost-plugin-incident-response/pull/136), although it looks like it will ship in v0.5.

Reply:
Thanks, I've noted that down for future iteration. :)